test/test_app_status.rb

```diff
@@ -0,0 +1,98 @@
+require_relative '../groups_admin'
+
+require_relative 'test_helper'
+
+class StatusAppGroupsTest < Minitest::Test
+
+  include Rack::Test::Methods
+  include TestHelper
+
+  def app
+    Sinatra::Application
+  end
+
+  TEST_DOMAIN='discussions-dev.its.umich.edu'
```

Comment:
the "TEST_DOMAIN" value is defined in couple of places. Can those definitions be combined?